www/tynk00/sp/resources/views/livewire/project-task-manager.blade.php

```diff
@@ -0,0 +1,168 @@
+        <a href="#" class="btn btn-light" data-toggle="modal" data-target="#addTaskModal">Nový úkol</a>
+        <div class="float-right">
+            <input type="checkbox" class="btn-check" id="btn-check-outlined" autocomplete="off" wire:model="closedTasks">
+            <label class="btn btn-outline-light" for="btn-check-outlined">Zobrazit dokončené úkoly</label><br>
+
+        </div>
+    </div>
+
+
+
+
+
+    <table class="table table-striped mb-0">
+        <thead>
+            <tr>
+                <th class="text-end text-nowrap" style="width: 50px;"></th>
+                <th>Name</th>
+                <th>Project</th>
+                <th>Due Date</th>
+                <th class="text-end text-nowrap" style="width: 50px;"></th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach ($tasks as $task)
+                <tr class="{{ $task->completed ? 'completed' : '' }}">
+                    <td>
+                        <div class="form-check">
+                            <input type="checkbox" class="custom-control-input
+                            name="completed" {{ $task->completed ? 'checked' : '' }}
+                            id="task{{ $task->id }}Checkbox" wire:click="completeTask({{ $task->id }})">
+                            <label class="custom-control-label" for="task{{ $task->id }}Checkbox"></label>
+                        </div>
+
+
+                    </td>
+                    <td>{{ $task->name }}</td>
+                    <td>
+                        @if ($task->project != null)
+                            {{ $task->project->name }}
+                        @endif
+                    </td>
+                    <td>{{ $task->due }}</td>
+
+                    <td class="text-end" style="width: 50px;">
+                        <div class="dropdown">
+                            <button class="btn btn-sm btn-link dropdown-toggle" type="button"
+                                id="task{{ $task->id }}Actions" data-bs-toggle="dropdown" aria-expanded="false">
+                            </button>
+                            <ul class="dropdown-menu" aria-labelledby="task{{ $task->id }}Actions">
+                                <li><a class="dropdown-item" href="{{ route('tasks.show', $task->id) }}">Zobrazit</a>
+                                </li>
+                                <li><a class="dropdown-item" data-toggle="modal" data-target="#editTaskModal"
+                                        wire:click="updateTask({{ $task->id }})">Upravit</a>
+                                </li>
+                                <form action="{{ route('tasks.destroy', $task->id) }}" method="POST">
+                                    @csrf
+                                    @method('DELETE')
+                                    <li><button type="submit" class="dropdown-item"
+                                            href="#">Odstranit</a></button>
```

Comment:
- melo by byt s potvrzenim
- 404 not found

Reply:
Přidáno potvrzení a udělal jsem další úpravy stránky s úkoly